plugins/toolbox-search/test/index.ts

```diff
@@ -27,7 +27,7 @@ function createWorkspace(
 }
 
 document.addEventListener('DOMContentLoaded', function () {
-  const toolbox = Object.assign({}, toolboxCategories);
+  const toolbox = Object.assign({}, toolboxCategories) as unknown as Blockly.utils.toolbox.ToolboxInfo;
```

Comment:
So it looks like the [type in the dev-tools plugin](https://github.com/google/blockly-samples/blob/6d1d5a57c78c2072f4874644a6eddca5f5764b35/plugins/dev-tools/src/index.d.ts#L112) is incorrect. Could you change `toolboxCategories` and `toolboxSimple` to both be of type `Blockly.utils.toolbox.ToolboxInfo`? Then you should be able to change this to:
```suggestion
  const toolbox = {...toolboxCategories};
```
And the types should work correctly!

Reply:
done!